### DISCUSSION TITLE
Next.js 13 app beta compatibility

### DISCUSSION THREAD

**Discussion by bombillazo** [nandorojo/solito#306] [Q&A] [answered]
Hello, can solito run a Next project using the Next.js app configuration? My worry is regarding the new Server Routing breaking the app, but we want to leverage the layouts and route grouping.

**Comment by nandorojo** [comment, marked as answer]
I don’t think so. The routing has many breaking changes. For example, there’s no hook to get the URL params from dynamic routes. I’ll revisit when it’s stable.
